mod-invoice-storage/schemas/invoice_line.json

```diff
@@ -8,6 +8,14 @@
       "type": "string",
       "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
     },
+    "accountingCode": {
+      "description": "Number that represents the vendor is an external accounting system which may include details like \"Address code\" in it. This is the number as stated on the invoice",
```

Comment:
> This is the number as stated on the invoice

This description doesn't seem to match the interface fields sheet, which says that the value is pulled from accountNumber which is inturn pulled from either POL/ based on selected vendor. Either way probably doesn't come from invoice